src/core/Akka.Remote/Transport/DotNetty/DotNettyTransportSettings.cs

```diff
@@ -0,0 +1,185 @@
+                maxFrameSize: ToNullableInt(config.GetByteSize("maximum-frame-size")) ?? 128000,
+                ssl: config.HasPath("ssl") ? SslSettings.Create(config.GetConfig("ssl")) : SslSettings.Empty,
+                dnsUseIpv6: config.GetBoolean("dns-use-ipv6", false),
+                tcpReuseAddr: config.GetBoolean("tcp-reuse-addr", true),
+                tcpKeepAlive: config.GetBoolean("tcp-keepalive", true),
+                tcpNoDelay: config.GetBoolean("tcp-nodelay", true),
+                backlog: config.GetInt("backlog", 4096),
+                enforceIpFamily: RuntimeDetector.IsMono || config.GetBoolean("enforce-ip-family", false),
+                receiveBufferSize: ToNullableInt(config.GetByteSize("receive-buffer-size") ?? 256000),
+                sendBufferSize: ToNullableInt(config.GetByteSize("send-buffer-size") ?? 256000),
+                writeBufferHighWaterMark: ToNullableInt(config.GetByteSize("write-buffer-high-water-mark")),
+                writeBufferLowWaterMark: ToNullableInt(config.GetByteSize("write-buffer-low-water-mark")),
+                backwardsCompatibilityModeEnabled: config.GetBoolean("enable-backwards-compatibility", false),
+                logTransport: config.HasPath("log-transport") && config.GetBoolean("log-transport"),
+                byteOrder: order);
+        }
+
+        private static int? ToNullableInt(long? value) => value.HasValue && value.Value > 0 ? (int?)value.Value : null;
+
+        private static int ComputeWorkerPoolSize(Config config)
+        {
+            if (config == null) return ThreadPoolConfig.ScaledPoolSize(2, 1.0, 2);
+
+            return ThreadPoolConfig.ScaledPoolSize(
+                floor: config.GetInt("pool-size-min"),
+                scalar: config.GetDouble("pool-size-factor"),
+                ceiling: config.GetInt("pool-size-max"));
+        }
+
+        public readonly TransportMode TransportMode;
+        public readonly bool EnableSsl;
+        public readonly TimeSpan ConnectTimeout;
+        public readonly string Hostname;
+        public readonly string PublicHostname;
+        public readonly int Port;
+        public readonly int ServerSocketWorkerPoolSize;
+        public readonly int ClientSocketWorkerPoolSize;
+        public readonly int MaxFrameSize;
+        public readonly SslSettings Ssl;
+        public readonly bool DnsUseIpv6;
+        public readonly bool TcpReuseAddr;
+        public readonly bool TcpKeepAlive;
+        public readonly bool TcpNoDelay;
+        public readonly bool EnforceIpFamily;
+        public readonly int Backlog;
+        public readonly int? ReceiveBufferSize;
+        public readonly int? SendBufferSize;
+        public readonly int? WriteBufferHighWaterMark;
+        public readonly int? WriteBufferLowWaterMark;
+        public readonly bool BackwardsCompatibilityModeEnabled;
+        public readonly bool LogTransport;
```

Comment:
This is for activating that Logging `ChannelHandlerAdapter` from earlier, right?